### PR TITLE
Feature/remove

### PR DESCRIPTION
**Pull request by chack05**
* removes _id and _rev when returning data
* uses _id hash as Zone-id and _id to store the Message-id

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -167,10 +181,45 @@ server.post('/api/addmessages', function(req, res) {
...
+                } else {
+                    res.status(404).send('Database error! Couldn\'t fetch ID check messages: ' + err);
+                }
+            }
+        );
+        if(idExists !== "") break;
+    }
+
+    if (idExists !== "") {
+        res.status(404).send('ID already existing: ' + idExists);
```
Comment (JanVan01) [inline index.js]: This is going to happen a lot. Each of the clients will push the whole list of messages they have there is going to be duplicates

**Comment by chack05** [comment]
@JanVan01 Do you suggest to simply drop duplicates?

**Comment by JanVan01** [comment]
@chack05 Yes, if all client ids are unique we can just drop duplicates because they are indeed only duplicates.

File: setup_database/install.js [hunk 2]
```diff
@@ -15,6 +15,9 @@ var messagedesign = {
     "views": {
         "by_zoneid_and_date": {
            "map": "function(doc){ emit([doc['Zone-id'],doc['Expired-at']], doc)}"
```
Comment (JanVan01) [inline setup_database/install.js]: Can't that be removed? Is there still a Zone-ID?

Comment (chack05) [inline setup_database/install.js]: Well, we still pass a zone parameter to the message request, so I think
it's still required, I guess. It's just that Zone-id now stores the _id
from the zone. Or do I miss some detail?

**Pull request merged**